### PR TITLE
The Ark of the Clockwork Justicar will now force objects away from it

### PR DESCRIPTION
**Pull request by ChangelingRain**
:cl: Joan
rscadd: The Ark of the Clockwork Justicar will now force objects away from it.
/:cl:

This ensures it'll take up a 3x3 area without actually being 3x3; any attempts at barricading it in with objects immediately next to it will be rejected.
Do take note you can still barricade AROUND that 3x3 area, however.

### CODE REVIEW

File: code/modules/atmospherics/machinery/portable/portable_atmospherics.dm [hunk 1]
```diff
@@ -62,6 +62,11 @@
 	anchored = 1 //Prevent movement
 	return 1
 
+/obj/machinery/portable_atmospherics/Move()
```
Comment (Iamgoofball) [inline code/modules/atmospherics/machinery/portable/portable_atmospherics.dm]: we all know how atmos is super finicky, please make sure this meme doesn't result in some stupid things happening

Comment (Crushtoe) [inline code/modules/atmospherics/machinery/portable/portable_atmospherics.dm]: inb4 potato cannon atmos

**Comment by coiax** [comment]
Why not just make it 3x3

**Comment by ChangelingRain** [comment]
Because nothing handles large objects properly that's why

**Pull request merged**